docs/messaging/messaging-metrics.md

```diff
@@ -1,20 +1,18 @@
-# Semantic Conventions for Messaging Metrics
+# Semantic Conventions for Messaging Client Metrics
 
 **Status**: [Experimental][DocumentStatus]
 
 <!-- Re-generate TOC with `markdown-toc --no-first-h1 -i` -->
 
 <!-- toc -->
 
-- [Common attributes](#common-attributes)
+- [Common metrics](#common-metrics)
+  - [Metric: `messaging.client.request.duration`](#metric-messagingclientrequestduration)
```

Comment:
Why using `messaging.client.request.duration` instead of `messaging.client.operation.duration` (and be consistent with DB metrics)?

Reply:
I want to make the difference between operations that application initiates and processing obvious in the metric name.

Processing is also an operation, so someone might try to look for `process` operation in `messaging.client.operation.duration`. 

Names like `messaging.client.request|call.duration` would make it obvious.

While databases do support change-feeds and may have process-like operation, it's not the key scenario and database semconv does not really have this problem.

Reply:
Ok, this makes sense (and is consistent with HTTP conventions).

> Processing is also an operation, so someone might try to look for `process` operation in `messaging.client.operation.duration`.

On the other hand, someone might not expect `create` operations in `messaging.client.request.*`, as it doesn't involve a broker request?

Reply:
> On the other hand, someone might not expect create operations in messaging.client.request.*, as it doesn't involve a broker request?

I'd say there is no point in reporting `create` duration since it's 0. So it checks out :) I can mention it explicitly.